### ISSUE TITLE
[Question] On HACS/Integrations, the only TrueNAS integration I found is tomaae / homeassistant-truenas

### ISSUE THREAD

**Issue by janusn** [cyr-ius/hass-truenas#2]
First of all, thanks for your effort of forking the integration from tomaae. It will be really helpful.

However, I cannot find your version of the hass-truenas integration. When I searched "truenas" in HACS integrations, the only integration shows up was tomaae's.

<img width="890" alt="Screen Shot 2024-02-01 at 6 01 43 pm" src="https://github.com/cyr-ius/hass-truenas/assets/21253783/9e3c4597-faea-4a67-99ae-4623c50495ca">

<img width="890" alt="Screen Shot 2024-02-01 at 6 00 26 pm" src="https://github.com/cyr-ius/hass-truenas/assets/21253783/a13a5eee-02dd-428d-9b58-cfaa9344d4e7">

Do I need to add a custom repository in order to find it? And can this integration has a distinctive name? There are 2 integrations for HASS named as "TrueNAS" already. It is pretty confusing even before yours comes along.

**Comment by googanhiem** [comment]
Think you'll just have to copy the `truenas` folder into the `custom_components` folder in your home assistant `config` folder (what we did before HACS). Just remember to delete the other truenas folder and device settings (in HA), as they will conflict.

source: Thats what I did.

It mostly works and has a couple extra features, but I believe this component has the same issues as Tomaee, some of the API calls are broken due to Truenas updating to Cobia (I want my arc/memory stats back in hass).
